concrete/src/Validator/ValidatorManagerInterface.php

```diff
@@ -1,23 +1,21 @@
 <?php
+
 namespace Concrete\Core\Validator;
 
 /**
- * Interface ValidatorManagerInterface
  * A generic validator manager interface that enables validating against many validators at once.
- *
- * \@package Concrete\Core\Validator
  */
 interface ValidatorManagerInterface extends ValidatorInterface
 {
     /**
-     * Get a list of all validators.
+     * Get a list of all validators keyed by their handles.
      *
-     * @return ValidatorInterface[] Array of validators keyed by their handles
+     * @return \Concrete\Core\Validator\ValidatorInterface[]
      */
     public function getValidators();
 
     /**
-     * Does a validator with this handle exist.
+     * Does a validator with this handle exist?
```

Comment:
IMO these descriptors should always be statements IE "Check if a validator exists with a handle" vs "Does it exist?" but we don't have to hold up the PR for this.